src/fides/api/ops/api/v1/endpoints/dataset_endpoints.py

```diff
@@ -193,12 +195,23 @@ def patch_dataset_configs(
             .filter_by(fides_key=dataset_pair.ctl_dataset_fides_key)
             .first()
         )
-        fetched_dataset: Dataset = Dataset.from_orm(ctl_dataset)
+        if not ctl_dataset:
+            raise HTTPException(
+                status_code=HTTP_404_NOT_FOUND,
+                detail=f"No ctl dataset with key '{dataset_pair.ctl_dataset_fides_key}'",
+            )
+
+        try:
+            fetched_dataset: Dataset = Dataset.from_orm(ctl_dataset)
```

Comment:
A lot of existing ops datasets don't have an `organization_fides_key` or a `data_qualifier`, but they are not optional fields on the Dataset model.  If a ctl dataset has been created from an ops dataset, these values will be None, and then throw a validation error when parsed here.

This change is to prevent a 500 Internal server error, and catch and throw a 422 instead.  The user should then return to the ctl_dataset and add the organization_fides_key and data_qualifier.

Reply:
should we make a 🎫 for this issue?

Reply:
yep will do!

Reply:
https://github.com/ethyca/fides/issues/2115